src/json-patch-duplex.js

```diff
@@ -587,7 +617,8 @@ var jsonpatch;
      * or just be undefined
      * @deprecated
      */
-    function apply(document, patch, validateOperation) {
+    function apply(document, patch, validateOperation, copyByValue) {
```

Comment:
Imo deprecated API should be feature-locked - `mutateDocument` is not configurable either